python/paddle/fluid/__init__.py

```diff
@@ -113,18 +113,17 @@ def __bootstrap__():
         'use_pinned_memory', 'check_nan_inf', 'benchmark', 'warpctc_dir',
         'eager_delete_scope', 'use_mkldnn', 'initial_cpu_memory_in_mb',
         'init_allocated_mem', 'free_idle_memory', 'paddle_num_threads',
-        'dist_threadpool_size', 'cpu_deterministic', 'eager_delete_tensor_gb',
-        'reader_queue_speed_test_mode'
+        "dist_threadpool_size", 'cpu_deterministic', 'eager_delete_tensor_gb'
```

Comment:
Why reader_queue_speed_test_mode  is deleted?

Reply:
Revert code. Thanks!